src/main/java/org/gitlab4j/api/GroupApi.java

```diff
@@ -224,6 +224,46 @@ public GroupApi(GitLabApi gitLabApi) {
         return (new Pager<Group>(this, Group.class, itemsPerPage, formData.asMap(), "groups", groupId, "subgroups"));
     }
 
+    /**
+     * Get a list of projects belonging to the specified group ID.
+     *
+     * GET /groups/:id/projects
+     *
+     * @param groupId the group ID to list the projects for
+     * @param archived Limit by archived status
+     * @param visibility Limit by visibility public, internal, or private 
+     * @param orderBy Return projects ordered by id, name, path, created_at, updated_at, or last_activity_at fields. Default is created_at
+     * @param sortOrder Return projects sorted in asc or desc order. Default is desc
+     * @param search Return list of authorized projects matching the search criteria
+     * @param simple Return only the ID, URL, name, and path of each project
+     * @param owned Limit by projects owned by the current user
+     * @param starred Limit by projects starred by the current user
+     * @param withIssuesEnabled Limit by enabled issues feature
+     * @param withMergeRequestsEnabled Limit by enabled merge requests feature
+     * @param withCustomAttributes Include custom attributes in response (admins only)
+     * @return a list of projects belonging to the specified group ID
+     * @throws GitLabApiException if any exception occurs
+     */
+  public List<Project> getProjects(int groupId, Boolean archived, Visibility visibility, GroupProjectOrderBy orderBy,
+      SortOrder sortOrder, String search, Boolean simple, Boolean owned, Boolean starred, Boolean withIssuesEnabled,
+      Boolean withMergeRequestsEnabled, Boolean withCustomAttributes) throws GitLabApiException {
+      Form formData = new GitLabApiForm()
+          .withParam("archived", archived)
+          .withParam("visibility", visibility)
+          .withParam("order_by", orderBy)
+          .withParam("order_by", orderBy)
+          .withParam("sort", sortOrder)
+          .withParam("search", search)
+          .withParam("simple", simple)
+          .withParam("owned", owned)
+          .withParam("starred", starred)
+          .withParam("with_issues_enabled", withIssuesEnabled)
+          .withParam("with_merge_requests_enabled", withMergeRequestsEnabled)
+          .withParam("with_custom_attributes", withCustomAttributes);
+        Response response = get(Response.Status.OK, formData.asMap(), "groups", groupId, "projects");
+        return (response.readEntity(new GenericType<List<Project>>() {}));
+    }
```

Comment:
For all new methods that take this many properties, this library is now using the Filter pattern.  See the ```org.gitlab4j.api.models.ProjectFilter``` and its use in the following methods in the ProjectApi:
```
public List<Project> getUserProjects(Object userIdOrUsername, ProjectFilter filter)
public List<Project> getUserProjects(Object userIdOrUsername, ProjectFilter filter, int page, int perPage)
public Pager<Project> getUserProjects(Object userIdOrUsername, ProjectFilter filter, int itemsPerPage)
```

Reply:
Oh, that's better!

Reply:
Did you consider to use [Lombok](https://projectlombok.org/) ? Then you could simply get the `withXXX` for free by using [@Setter](https://projectlombok.org/features/GetterSetter)?

Reply:
Lombok is great, however for a library like this,  it is better to be explicit, specifically for documentation (Javadoc)  purposes. 

We have experimented using Lombok, but getting the serialization/deserialization correct can also be problematic, especially with a library of this size.

Reply:
Oh okay... Interesting to know.